Userguide and addNote bugs (#179)
AY2021S1-CS2113-T13-1/tp: docs/UserGuide.md

💡 Each note has to have a **UNIQUE** title and it is **CASE-INSENSITIVE**. <br>
💡 One can choose to add a `TAG` or/and `ISPIN`, `ISARCHIVE`. These parameters are optional.<br>

> 💡 Set `ISPIN` to "true" if you want the note to be pinned and any other value for an unpinned note.  
> 💡 Set `ISARCHIVE` to "true" if you want the note to be archived and any other value for an unarchived note.

Reviewer: ```suggestion
💡 Set `ISPIN` to "true" if you want the note to be pinned. Any other input value will leave the note unpinned.  
💡 Set `ISARCHIVE` to "true" if you want the note to be archived. Any other input value will leave the note unarchived.
```
